packages/flutter/lib/src/material/theme_data.dart

```diff
@@ -182,9 +185,18 @@ class ThemeData extends Diagnosticable {
     hintColor ??= isDark ?  const Color(0x80FFFFFF) : const Color(0x8A000000);
     errorColor ??= Colors.red[700];
     inputDecorationTheme ??= const InputDecorationTheme();
-    iconTheme ??= isDark ? const IconThemeData(color: Colors.white) : const IconThemeData(color: Colors.black);
-    primaryIconTheme ??= primaryIsDark ? const IconThemeData(color: Colors.white) : const IconThemeData(color: Colors.black);
-    accentIconTheme ??= accentIsDark ? const IconThemeData(color: Colors.white) : const IconThemeData(color: Colors.black);
+    iconTheme ??= isDark ? const IconThemeData(color: Colors.white) : const IconThemeData(
+      color: Colors.black,
+      opacity: _kIconThemeDefaultColorOpacity,
```

Comment:
I'm afraid that is probably overachieving. I don't think we want to start applying 0.87 to the opacity of all icons that specify a color, even though doing so is probably what's intended by the latest spec (https://material.io/design/iconography/system-icons.html#color).  

There's a fine backwards compatibility line here.  The appearance of an app that has specified an icon color explicitly shouldn't change; an app that's picking up the default icon color can change.